packages/core/src/external/aws/app-config.ts

```diff
@@ -79,6 +86,34 @@ export async function getFeatureFlags(
   throw new MetriportError(`Failed to get Feature Flags`);
 }
 
+export async function getFeatureFlagValueStringArrayCxValues<
```

Comment:
This function name is _very_ hard to read.... 😅 I read it about 5 times and still trying to make sense of it.

Reply:
Remove `StringArray` as I guess cxIds *only* exist on these types of FFs.